### PR TITLE
Add documentation for WiLight Fan

### PR DESCRIPTION
**Pull request by leofig-rj**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/core/pull/39541
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
Please rebase on latest `next` branch and solve the merge conflict.

**Comment by leofig-rj** [comment]
Can you please confirm if this is how I should do it?

Thank you!

[image: image.png]

Em seg., 25 de jan. de 2021 às 10:04, Martin Hjelmare <
notifications@github.com> escreveu:

**Comment by frenck** [comment]
> Can you please confirm if this is how I should do it?

> Thank you!

What do you mean by this? Images via email do not work. Please login into GitHub to respond.

You need to perform a git rebase onto the upstream next branch.

**Comment by leofig-rj** [comment]
Done!

**Comment by frenck** [comment]
There are still unrelated commits in this PR:

image


If I'm correct, only the last 3 commits matter? The others are from previous PRs?

**Comment by leofig-rj** [comment]
Yes, you are correct. The others are from the previous PR.

**Comment by frenck** [comment]
ok, rebased with just the last 3 commits onto the upstream `next` branch.

Please, if you create a PR next time, create a separate feature branch to do work on, that saves a lot of issues 😉

File: source/_integrations/wilight.markdown [hunk 1]
```diff
@@ -16,8 +17,26 @@ The `wilight` integration is to integrate [WiLight](http://www.wilight.com.br) d
 
 There is currently support for the following device types within Home Assistant:
 
-- Light (WiLight model I-100 and I-102)
+- Fan (WiLight model V-104).
+
+- Light (WiLight model I-100, I-102 and I-107).
```
Comment (frenck) [inline source/_integrations/wilight.markdown]:
```suggestion
- Fan (WiLight model V-104).
- Light (WiLight model I-100, I-102 and I-107).
```

**Review by frenck — APPROVED** [review]
Thanks, @leofig-rj 👍

**Comment by leofig-rj** [comment]
Thank you!

**Pull request merged**